py3status/modules/uname.py

```diff
@@ -38,20 +38,14 @@ class Meta:
             ],
         }
 
-    def show_uname(self):
-        system, node, release, version, machine, processor = uname()
-        response = {
+    def uname(self):
+        keys = ['system', 'node', 'release', 'version', 'machine', 'processor']
+        full_text = self.py3.safe_format(self.format, dict(zip(keys, uname())))
+
+        return {
             'cached_until': self.py3.CACHE_FOREVER,
-            'full_text': self.py3.safe_format(
-                self.format,
-                dict(system=system,
-                     node=node,
-                     release=release,
-                     version=version,
-                     machine=machine,
-                     processor=processor))
+            'full_text': self.py3.safe_format(self.format, full_text)
```

Comment:
small glitch here since `full_text` is already generated by self.py3.safe_format

Reply:
Good catch. I'll fix that. Awaiting on other comments.